src/main/resources/swagger.api/inn-reach.yaml

```diff
@@ -134,6 +134,37 @@ paths:
       operationId: deleteLocation
       parameters:
         - $ref: '#/components/parameters/id'
+  /authentication:
```

Comment:
once we apply API interface generation from tags, your endpoint interface will be named incorrectly.
I suggest to add `tags` attribute right now (see sample in https://github.com/folio-org/mod-inn-reach/pull/20)